ssha/config.py

```diff
@@ -125,6 +129,35 @@ def get(key, default=None):
     return render(value)
 
 
+def generate_key():
+    """
+    Generates and adds a key to the SSH agent.
+
+    """
+    key = rsa.generate_private_key(
+        backend=crypto_default_backend(),
+        public_exponent=65537,
+        key_size=4096
+    )
+    private_key = key.private_bytes(
+        crypto_serialization.Encoding.PEM,
+        crypto_serialization.PrivateFormat.PKCS8,
+        crypto_serialization.NoEncryption())
```

Comment:
Please move the closing bracket to its own line and ensure each argument line has a trailing comma, e.g.

```
thing = stuff(
    one,
    two,
    three,
)
```